src/xdr/xdr_codec.rs

```diff
@@ -210,16 +210,17 @@ impl<T: XdrCodec> XdrCodec for Box<T> {
 ///
 /// Basic usage:
 ///
-/// ```
-/// use crate::types::Auth;
+#[cfg_attr(feature = "all-types", doc = "```")]
+#[cfg_attr(not(feature = "all-types"), doc = "```ignore")]
+/// use substrate_stellar_sdk::{types::Auth, parse_stellar_type};
```

Comment:
Did you check the generated docs whether the lines
```
/// // For this test, we require the usage of `Auth` which is only compiled with
/// // the `all-types` feature.
/// // We will ignore the test if the feature is not enabled.
```

are ignored? 🤔

Reply:
I misunderstood what you meant in the first comment, they are not ignored (for that we would need # apparently), but since the comment is outside the scope of the doc test it does not show. Is this what you mean?
<img width="1001" alt="image" src="https://github.com/pendulum-chain/substrate-stellar-sdk/assets/96739519/09935e36-59e7-49ef-9fd7-7fb9354faa2b">

Reply:
Oh okay I misunderstood what you meant, I thought ignored on compilation. Latest push should show it like this (they just need to be in the scope of the test definition)
<img width="1001" alt="image" src="https://github.com/pendulum-chain/substrate-stellar-sdk/assets/96739519/5173a158-5a46-4114-b1a4-8399907e01ad">

Reply:
Nice, thanks 🙏